code/modules/surgery/organs/organ_external.dm

```diff
@@ -680,6 +684,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 		tbrute = 3
 	return "[tbrute][tburn]"
 
+/obj/item/organ/external/proc/update_splints() //we're processing splints like this instead of on a per step basis for efficiency
```

Comment:
imagine if you stood still.
and just did nothing.

Logically what would you expect to happen?
A. An effect related to moving to trigger despite you not moving
B. Nothing much

Doing things every step isn't great, no, but it's far better than continually checking if you've moved a step.
At /tg/ we have all worn, all held and all organ items informed that the mob moved, and it's no major impact.